### PR TITLE
Bug 2057495: Auto increase volume size to 20 GiB in the default storage class

### PR DESCRIPTION
**Pull request by jsafrane**
This will re-create the default storage class with a new `volumeSizeAutoAvailable` parameter that tells the driver to bump the volume sizes to 20 GiB. 

Users may opt-out by creating a different default storage class.

cc @storage

### CODE REVIEW

**Comment by jsafrane** [comment]
/hold
https://github.com/openshift/alibaba-cloud-csi-driver/pull/12 must be merged first

**Comment by jsafrane** [comment]
Umm, how the tests could pass if the driver does not support `volumeSizeAutoAvailable` yet?

**Comment by jsafrane** [comment]
There is an open BZ for that: https://bugzilla.redhat.com/show_bug.cgi?id=2047735 :-D

File: assets/storageclass.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ metadata:
 provisioner: diskplugin.csi.alibabacloud.com
 parameters:
   type: available
+  volumeSizeAutoAvailable: "true"
```
[the remark this answers is not in the document]
Comment (jsafrane) [reply assets/storageclass.yaml:10]: /hold
for a controller that can re-create the storage class

[the remark this answers is not in the document]
Comment (kwoodson) [reply assets/storageclass.yaml:10]: @jsafrane Any update on this? Anything I can do to assist?

[the remark this answers is not in the document]
Comment (jsafrane) [reply assets/storageclass.yaml:10]: We're waiting for https://github.com/openshift/library-go/pull/1348 and its 4.10 backport

[the remark this answers is not in the document]
Comment (jsafrane) [reply assets/storageclass.yaml:10]: I updated library-go with the new StorageClass controller, @bertinatto PTAL.

**Review by bertinatto — COMMENTED** [review]
/lgtm

**Comment by jsafrane** [comment]
/retest
It works for me on cmdline

**Comment by jsafrane** [comment]
/hold cancel
https://github.com/openshift/alibaba-cloud-csi-driver/pull/12 was merged

**Pull request merged**